main.js

```diff
@@ -0,0 +1,334 @@
+const apiKey = "6427275c4ee8b157888fdf144b2fc5ca";
```

Comment:
usually you don't put keys in github you would read it from an ENV variable but I'm sure your instructor will go over this.

Reply:
Since it's a free api subscription, they told me I could include. Haven't tackled node yet.